addons/mail/static/src/discuss/core/common/attachment_upload_service_patch.js

```diff
@@ -0,0 +1,24 @@
+/* @odoo-module */
+
+import { AttachmentUploadService } from "@mail/core/common/attachment_upload_service";
+
+import { patch } from "@web/core/utils/patch";
+
+patch(AttachmentUploadService.prototype, {
+    setup() {
+        super.setup(...arguments);
+        this.env.services["bus_service"].subscribe("mail.record/insert", ({ Thread }) => {
+            if (
```

Comment:
note: type of this patch is currently not detected, completion doesn't work inside it